xml/ssh_banners.xml

```diff
@@ -381,9 +392,10 @@ fingerprint SSH servers.
     <param pos="0" name="os.product" value="Linux"/>
     <param pos="0" name="os.version" value="8.0"/>
   </fingerprint>
-  <fingerprint pattern="^OpenSSH_([^\s]+)\s+(Debian.+squeeze.*)$">
+  <fingerprint pattern="^OpenSSH_([^\s]+)\s+(Debian.+[-6|squeeze].*)$">
```

Comment:
A yes the `.+` on the left side is too greedy. I like your recommendation, I think that's the best way to make the `squeeze` optional.